hadoop-hdfs-project/hadoop-hdfs-client/src/main/java/org/apache/hadoop/hdfs/protocol/datatransfer/PipelineAck.java

```diff
@@ -230,14 +260,27 @@ public static ECN getECNFromHeader(int header) {
     return StatusFormat.getECN(header);
   }
 
+  public static SLOW getSLOWFromHeader(int header) {
+    return StatusFormat.getSLOW(header);
+  }
+
   public static int setStatusForHeader(int old, Status status) {
     return StatusFormat.setStatus(old, status);
   }
 
+  public static int setSLOWForHeader(int old, SLOW slow) {
```

Comment:
Only the unit test uses this method. Would you please add VisibleForTesting?
```suggestion
  @VisibleForTesting
  public static int setSLOWForHeader(int old, SLOW slow) {
```